test-files/cmd/glacial/ServiceOsmApiDbDcConflateTest.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+test-files/cmd/glacial/OsmApiDbConflate.off test-files/DcGisRoads.osm test-files/DcTigerRoads.osm -77.040,38.892,-77.033,38.896 ServiceOsmApiDbDcConflateTest
```

Comment:
Do you really need to run w/ DcGis? It is a bit slow and `./test-files/conflate/unified/AllDataTypes*.osm` is probably a better test.